### ISSUE TITLE
Multiplayer image on mod detail pages does not have alt text. 

### ISSUE THREAD

**Issue by BaineGames** [satisfactorymodding/smr-frontend#16]

**Comment by CrowhopTech** [comment]
Hi @BaineGames! I'm going through and taking on a bunch of minor tasks (thanks so much for your other issues btw, I've started working on some of them).

I can't find the "multiplayer image" you're talking about on the mod detail page you refer to here. It may just be I haven't looked at a good example mod, or maybe it's since been removed, but could you point me to an example of this (maybe a screenshot of where it is on the page)?

**Comment by CrowhopTech** [comment]
Oh wait! Right after I sent that I found what you were referring to 🙃

**Comment by budak7273** [comment]
Turns out this is an image some modders into their Long Descriptions to indicate mod compatibility, not a platform feature. So, closing in favor of #17 

https://github.com/deantendo/community
